Apps/W1/EDocumentConnectors/Tietoevry/app/src/Permissions/TietoevryEDocConnectorRead.PermissionSetExt.al

```diff
@@ -0,0 +1,12 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.EServices.EDocumentConnector.Tietoevry;
+
+using System.Security.AccessControl;
+
+permissionsetextension 6392 "Tietoevry EDoc. Connector - Read" extends "D365 READ"
```

Comment:
Same

Reply:
Not sure what you mean, forked from your Avalara Connector.